### ISSUE TITLE
Cannot achieve the comparable accuracy results with the provided

### ISSUE THREAD

**Issue by EddieEduardo** [weiaicunzai/pytorch-cifar100#69]
Hello, thanks for code sharing.

I use Cifar100 to train and test the ShuffleNetV2, but the accuracy I got is just 0.499, all the parameter setting is consistent with the provided in **READMe**, which are,  init lr = 0.1 divide by 5 at 60th, 120th, 160th epochs, train for 200 epochs with batchsize 128 and weight decay 5e-4, Nesterov momentum of 0.9 (I have run 2 times on my end and 0.499 is the best accuracy). 

Looking forword to anyone's reply, thanks a lot again!

**Comment by EddieEduardo** [comment]
Plus, my PyTorch version is 1.8.0.

**Comment by LK-WordSnake** [comment]
After repeated experiments, this code fails to achieve the described accuracy in the CIFAR100.
You can set a higher input size for training, which can achieve a higher accuracy.

**Comment by hayleeXinyi** [comment]
For densenet161, Top 1 err:  tensor(0.2350) Top 5 err:  tensor(0.0520)
